### PR TITLE
fix

### PR DESCRIPTION
**Pull request by zdubzdab**

### CODE REVIEW

File: app/assets/stylesheets/bootstrap_and_overrides.css.less [hunk 1]
```diff
@@ -193,3 +193,9 @@ footer {
   position:relative;
   left:75%;
 }
+
+
+.pagination {
+  font-size: 12px;
+  font-weight: bold;
+}
```
Comment (begin29) [inline app/assets/stylesheets/bootstrap_and_overrides.css.less]: add config to sublime that create new empty line for saved file. 
```
	"ensure_newline_at_eof_on_save": true
```

File: app/assets/stylesheets/welcome.scss [hunk 2]
```diff
@@ -140,6 +127,33 @@
   }
 }
 
+#right-panel {
+  position:relative;
+  left: -0.6%;
```
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: if you want to create margin between 2 elements, please use 
```css
margin-left or margin-right: few pixels
```
percents here is not useful.

File: app/assets/stylesheets/welcome.scss [hunk 3]
```diff
@@ -140,6 +127,33 @@
...
+  top: 7px;
+  width: 34%;
+}
+
+#tiding_list {
+  position:relative;
+  top: 15px;
+  left: -4%;
```
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: ^^

File: app/assets/stylesheets/welcome.scss [hunk 4]
```diff
@@ -140,6 +127,33 @@
...
+  width: 116%;
+  height: 309px;
+  margin-bottom: 72px;
+}
+
+#tiding_table_pagination {
+  width: 120%;
+  position:relative;
+  left: -2%;
```
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: ^^

File: app/assets/stylesheets/welcome.scss [hunk 5]
```diff
@@ -140,6 +127,33 @@
...
+}
+
+#list-tiding-headers {
+  position:relative;
+  left: -7%;
```
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: ^^

File: app/assets/stylesheets/welcome.scss (hunk 2 above)
Comment (begin29) [inline app/assets/stylesheets/welcome.scss]: if it is internal distance inside block, you can use `padding` option

File: app/assets/stylesheets/bootstrap_and_overrides.css.less (hunk 1 above)
Comment (zdubzdab) [inline app/assets/stylesheets/bootstrap_and_overrides.css.less]: I have this config

**Pull request merged**